client/src/components/secretMessage.test.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import {
+  render, screen,
+} from '@testing-library/react';
+import axios from 'axios';
+import SecretMessage from './SecretMessage';
+
+test('render the secret body when there is an valid id', async () => {
+  // Arrange
+  const fakeId = '';
+  const res = {
+    status: 200,
+    data: {
+      id: fakeId,
+      body: 'test',
+    },
+  };
+  // Act
+  jest.spyOn(axios, 'get').mockImplementation(() => Promise.resolve(res));
+  render(<SecretMessage id={fakeId} />);
+  // Assert
+  const element = await screen.findByText(/test/);
```

Comment:
Has this must be case-sensitive?

```suggestion
  const element = await screen.findByText(/test/i);
```

Reply:
Then why is this one shouldn't be case-sensitive? Every letter should be exactly the same.

Reply:
This was a question. Not every `Suggested change` is a recommendation. If `Every letter should be exactly the same.`, let's keep it as it was.